homeassistant/components/media_player/__init__.py

```diff
@@ -905,7 +906,11 @@ async def async_toggle(self) -> None:
             )
             return
 
-        if self.state in (STATE_OFF, STATE_IDLE, STATE_STANDBY):
+        if self.state in (
+            MediaPlayerState.OFF,
+            MediaPlayerState.IDLE,
+            MediaPlayerState.STANDBY,
+        ):
```

Comment:
```suggestion
        if self.state in {
            MediaPlayerState.OFF,
            MediaPlayerState.IDLE,
            MediaPlayerState.STANDBY,
        }:
```
A set is slightly better